composer.json

```diff
@@ -71,15 +76,11 @@
         "antidot-fw/doctrine",
         "antidot-fw/message-queue",
         "antidot-fw/session",
-        "antidot-fw/aura-router-adapter",
         "antidot-fw/cli",
-        "antidot-fw/fast-router-adapter",
-        "antidot-fw/phug-te,plate-renderer",
+        "antidot-fw/phug-template-renderer",
         "antidot-fw/event-dispatcher",
         "antidot-fw/symfony-config-translator",
-        "wshafer/psr11-monolog",
-        "laminas/laminas-httphandlerrunner",
-        "laminas/laminas-diactoros"
+        "wshafer/psr11-monolog"
       ]
     }
   }
```

Comment:
Should we need to add the Composer script about running development server easily?

And I think it can let [Quick Start](https://github.com/antidot-framework/antidot-starter/blob/53626c530aa7a2a7a8ed28908a680f61eb9e0702/README.md#quick-start) be a Composer script.

Reply:
It could be something like:

```
composer dev-server
```

Reply:
According to this [composer.json script](https://github.com/antidot-framework/antidot-starter/blob/3.x.x/composer.json#L51), I think we can define and use `composer run dev-server` to complete this issue.

Reply:
We should document this:

![image](https://user-images.githubusercontent.com/1093654/144295917-41576a61-2bb0-41f1-8986-f4434c3f3b4e.png)

You can run default host by running:

```
composer run dev-server
```

Or run it in custom host by

```
APP_HOST=127.0.0.1:8080 composer run dev-server
```